core/kernel_init.c

```diff
@@ -82,25 +82,25 @@ static void *idle_thread(void *arg)
     return NULL;
 }
 
-const char *main_name = "main";
-const char *idle_name = "idle";
+const char  *main_name  = "main";
+const char  *idle_name  = "idle";
 
 static char main_stack[THREAD_STACKSIZE_MAIN];
 static char idle_stack[THREAD_STACKSIZE_IDLE];
 
 void kernel_init(void)
 {
-    (void) disableIRQ();
+    (void)disableIRQ();
```

Comment:
Can't we remove the `(void)` completely? Looks like a leftover from a try to make core MISRA compliant.